tools/depends/target/python3/Makefile

```diff
@@ -41,7 +41,8 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	cd $(PLATFORM); patch -p1 -i ../android-nl_langinfo.patch
 	cd $(PLATFORM); patch -p1 -i ../apple.patch
 ifeq ($(OS),darwin_embedded)
-	cd $(PLATFORM); patch -p1 -i ../make-fork-optional.patch
+	cd $(PLATFORM); patch -p1 -i ../darwin_embedded.patch
+	cd $(PLATFORM); sed -ie 's|MACHDEP="unknown"|MACHDEP="darwin"|' configure.ac
```

Comment:
What's this needed for?

Reply:
we actually use the darwin platform as a code path. ios isnt detected as a platform (because it isnt in python), so we can either piggyback off darwin, or we implement the ios platform and then ifdefs go nuts through the code base

Reply:
Please check if it is really needed in the current state of feature_python3 branch.

Reply:
ok, running up a build now

Reply:
seems ok from my limited testing. Have pushed update